calendar_resource/README.rst

```diff
@@ -10,11 +10,26 @@ This module adds new features to facilitate resource management with meetings:
 
 * To attach resources to a meeting, go to `Calendar` => Click on a meeting => `Edit` =>
   `Edit` => add any related resources.
-* To edit or add a resource, go to `Settings` => `Resource` => `Resources`.
+* To edit or add a resource, go to `Settings` => `Resource` => `Resources`. Developer mode
+  has to be enabled for this menu to show.
```

Comment:
Hmmm it seems we have a problem here - the Calendar in v10 doesn't have a sidebar so it's not possible to add menu items:

![image](https://user-images.githubusercontent.com/7397578/29226317-635f2dea-7e86-11e7-8ed8-fe20f3bef213.png)

Not really sure the best place in this instance 😦